daemon/execdriver/native/apparmor.go

```diff
@@ -40,15 +40,12 @@ profile {{.Name}} flags=(attach_disconnected,mediate_deleted) {
   file,
   umount,
 
-  signal (receive) peer=/usr/bin/docker,
-  signal (receive) peer=docker-unconfined,
-
   deny @{PROC}/sys/fs/** wklx,
   deny @{PROC}/fs/** wklx,
   deny @{PROC}/sysrq-trigger rwklx,
   deny @{PROC}/mem rwklx,
   deny @{PROC}/kmem rwklx,
-  deny @{PROC}/kore rwklx,
+  deny @{PROC}/kcore rwklx,
   deny @{PROC}/sys/kernel/[^s][^h][^m]* wklx,
   deny @{PROC}/sys/kernel/*/** wklx,
 
```

Comment:
We also need to remove the 'deny ptrace' line below. This also does not exist in AppArmor 2.7.